src/posts/gpu-pipeline/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: 'Accelerating AI/ML Workflows in Earth Sciences with GPU-Native Xarray and Zarr (and more!)'
+date: '2025-05-01'
+
+authors:
+  - name: Negin Sobhani
+    github: negin513
+  - name: Wei Ji Leong
+    github: weiji14
+  - name: Max Jones
+    github: maxjones
+  - name: Akshay Subranian
+    github: akshaysubr
+  - name: Thomas Augspurger
+    github: tomaugspurger
+  - name: Katelyn Fitzgerald
+    github: kafitzgerald
+
+summary: 'How to accelerate AI/ML workflows in Earth Sciences with GPU-native Xarray and Zarr.'
+---
+
+# Accelerating AI/ML Workflows in Earth Sciences with GPU-Native Xarray and Zarr (and more!)
+
+## Introduction
+
+In large-scale geospatial AI and machine learning workflows, data loading is often the main bottleneck. Traditional pipelines rely on CPUs to preprocess and transfer massive datasets from storage to GPU memory, consuming resources and limiting scalability and effective use of GPU resources.
+
+To tackle this issue, a team from the [National Center for Atmospheric Research (NSF-NCAR)](https://ncar.ucar.edu) and [Development Seed](https://developmentseed.org) with mentors from [NVIDIA](https://www.nvidia.com) participated in a GPU hackathon to demonstrate how AI/ML workflows in Earth system sciences can benefit from GPU-native workflows using tools such as [Zarr](https://zarr.readthedocs.io/), [KvikIO](https://docs.rapids.ai/api/kvikio/stable/), and [DALI](https://developer.nvidia.com/dali).
+
+In this post, we share our hackathon experience, the integration strategies we explored, and the performance gains we achieved to highlight how modern tools can transform data-intensive workflows.
```

Comment:
Let's call out the hackathon and its funders by name (useful in this _climate_ hehe)

Reply:
I see this at the end, but it would be nice to call it out up top

Reply:
Done. Thanks for the suggestion. 🥇

Reply:
100% agreed. Linked them at the beginning of the blog post and would be happy to highlight them more too. :-)